CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0 (2020-07-28)
+
+- Added a directive `skyAgGridDataManagerAdapter` to handle interaction between ag-Grid and SKY UX data manager. [#59](https://github.com/blackbaud/skyux-ag-grid/pull/59)
+- Added an example using ag-Grid with SKY UX data manager. [#61](https://github.com/blackbaud/skyux-ag-grid/pull/61)
```

Comment:
Same questions about whether "ag-Grid" should be "the data entry grid component" since that's how we refer to the component in the SKY UX docs and whether should "SKY UX data manager" be "the data manager component."